Bazzar/core-service/src/main/java/ru/bazzar/core/entities/Characteristic.java

```diff
@@ -28,14 +34,6 @@ public class Characteristic {
     @ManyToOne
     private Product product;
 
-    @Override
-    public String toString() {
-        return "Characteristic{" +
-                "id=" + id +
-                ", name='" + name + '\'' +
-                ", product=" + product +
-                '}';
-    }
 
     @Override
     public boolean equals(Object o) {
```

Comment:
https://thorben-janssen.com/ultimate-guide-to-implementing-equals-and-hashcode-with-hibernate/